src/insexpand.c

```diff
@@ -2437,7 +2437,7 @@ ins_compl_stop(int c, int prev_mode, int retval)
 	    int	    compl_len = get_compl_len();
 
 	    if ((int)plen > compl_len)
-		ins_compl_insert_bytes(p + compl_len, (int)(plen - compl_len));
+		ins_compl_insert_bytes(p + compl_len, (int)plen - compl_len);
```

Comment:
In the `if` condition it's `plen` that is cast to `int`, so doing that here is more consistent than implicitly casting `compl_len` to `size_t`.